site/scripts/dev.ts

```diff
@@ -0,0 +1,22 @@
+import chalk from "chalk";
+import execa from "execa";
+
+const script = async () => {
+  console.log(chalk.bold("Launching site in dev mode..."));
+
+  await (
+    await import("./generate-sitemap")
+  ).default;
+
+  await (
+    await import("./generate-blocks-data")
+  ).default;
+
+  await (
+    await import("./create-db-indexes")
+  ).default;
+
+  await execa("next", ["dev"], { stdio: "inherit" });
```

Comment:
This might not be an actual problem, but it seems that `execa` buffers `stdout` by default [seen here](https://github.com/sindresorhus/execa#maxbuffer). I am wondering if we are going to hit that limit realistically? Probably not, but worth keeping in mind.

Reply:
Good point! It’s 100MB by default though, so I hope it will be enough 😅

Alternatively, we can launch Next.js server here directly, i.e. in the same process. That would mean that we switched to a [custom server](https://nextjs.org/docs/advanced-features/custom-server) though – not sure if it’d be a good thing to do.

Reply:
Wonder what will happen at the buffer limit. I assume the buffer simply wraps around and still outputs to `stdout`, so perhaps it's a non-issue